container_images/tox/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Assembles multiple versions of Python into a single container.
+
+FROM python:3.5.9-slim-buster
```

Comment:
why do we need multiple versions of python?

Reply:
Regarding Tox, the benefits I've found:
- Test against specific, or multiple, versions of Python. I was previously burned when deploying a change that used Python features available on my workstation (Python 3.8), but were not available in the Debian 9 environment where the change was running (Python 3.5).
- Regarding multiple versions, another team is interested in using [1], but they run a different version of Python, so we'd want to test against both.
- The tests are jailed from the system's dependencies; so, if a dependency isn't explicitly modeled, the test will fail. 
- Support for test-only dependencies. In [1], pyyaml is only required for tests, but not at runtime. Setup tools has a deprecated feature `tests_require`, but are recommending users switch to a dedicated test runner such as 'tox'. [2]

1. https://github.com/EricEdens/compute-image-tools/tree/inspect-unit/daisy_workflows/image_import/inspection
2. https://setuptools.readthedocs.io/en/latest/setuptools.html